examples/cgo/cc_dependency/BUILD

```diff
@@ -0,0 +1,36 @@
+config_setting(
+    name = "darwin",
+    values = {"cpu": "darwin"},
+)
+
+cc_library(
+    name = "version",
+    srcs = ["cxx_version.cc"],
+    hdrs = ["version.h"],
+    # TODO(yugui) Support darwin too and remove this workaround.
+    # See also comments in cxx_version.cc.
```

Comment:
you can use [@bazel_tools//tools/cpp:osx_cc_wrapper.sh](https://github.com/bazelbuild/bazel/blob/master/tools/cpp/osx_cc_wrapper.sh) for wrapping calls to gcc for the link phase and it will replace the -Wl,-rpath command with the correct calls to otool.